exercises/practice/accumulate/.docs/instructions.append.md

```diff
@@ -0,0 +1,7 @@
+# Instructions append
+
+## Workflow
+
+The tests for this exercise will cause compile time errors,
```

Comment:
```suggestion
The tests for this exercise will cause compile time errors.
```

Reply:
The next line is actually part of the sentence, I put it all on one line.